.github/workflows/dependency-license-check.yml

```diff
@@ -14,6 +14,8 @@ jobs:
     if: (github.actor != 'dependabot[bot]')
     steps:
       - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
```

Comment:
Could be helpful to comment why we're using this value (i.e. a link to the git-auto-commmit-action README)

Reply:
Added